app/src/main/java/de/danoeh/antennapod/dialog/FeedFilterDialog.java

```diff
@@ -0,0 +1,41 @@
+package de.danoeh.antennapod.dialog;
+
+import android.content.Context;
+
+import androidx.appcompat.app.AlertDialog;
+
+import org.greenrobot.eventbus.EventBus;
+
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.core.event.UnreadItemsUpdateEvent;
+import de.danoeh.antennapod.core.preferences.UserPreferences;
+
+public class FeedFilterDialog {
+    public static void showDialog(Context context) {
+        AlertDialog.Builder dialog = new AlertDialog.Builder(context);
+        dialog.setTitle(context.getString(R.string.pref_filter_feed_title));
+        dialog.setPositiveButton(null, null);
```

Comment:
This looks a bit strange. Shouldn't the button either have a text or just not be there?

Reply:
Yes, this should hide the positive button. I do not know if this is needed, mostly I work with the library by afollestad.

Reply:
Just not writing this line should hide the button, too